policies.tf

```diff
@@ -81,6 +81,7 @@ data "aws_iam_policy_document" "task_container_secrets" {
     )
     actions = [
       "secretsmanager:GetSecretValue",
+      "ssm:GetParameters",
```

Comment:
This is only referencing secret arns from Secrets Manager. How can this work for parameters? Perhaps, add an input variable like `var.task_container_parameters` to speficy them?